jsonschema-validation.xml

```diff
@@ -172,15 +172,26 @@
 
             <section title="Validation of primitive types and child values">
                 <t>
-                    Two of the primitive types, array and object, allow for child values.  The validation of
-                    the primitive type is considered separately from the validation of child instances.
+                    Two of the primitive types, array and object, allow for child values.
+                    The validation of the primitive type is considered separately from
+                    the validation of child instances.
                 </t>
                 <t>
-                    For arrays, primitive type validation consists of validating restrictions on length.
+                    For arrays, primitive type validation consists of validating
+                    restrictions on length with "minItems" and "maxItems", while
+                    "items" and "additionalItems" control which subschemas apply
+                    to which elements of the array.  In addition, "uniqueItems"
```

Comment:
It also highlights how these keywords (like "allOf"/"anyOf"/"oneOf"/"not") are important for more than just validation.  All other vocabularies, such as hyper-schema, are pretty much useless without these keywords as they are the only way to apply other vocabularies to complex instances.  The core vocabulary ("$schema", "$id", "$ref") does not provide any such mechanism, and vocabularies such as hyper-schema should not re-implement the process.  These keywords and the logical keywords have implications far beyond validation.